specification/testbase/resource-manager/Microsoft.TestBase/preview/2023-11-01-preview/customimage.json

```diff
@@ -0,0 +1,1131 @@
+      "put": {
+        "tags": [
+          "Test Base Custom Image"
+        ],
+        "operationId": "CustomImages_Create",
+        "description": "Creates a test base custom image.",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "parameters": [
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CustomImageResource"
+            },
+            "description": "Parameters supplied to create a test base custom image."
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./testbase.json#/parameters/TestBaseAccountNameParameter"
+          },
+          {
+            "$ref": "#/parameters/CustomImageNameParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The test base custom image was successfully created or updated.",
+            "schema": {
+              "$ref": "#/definitions/CustomImageResource"
+            }
+          },
+          "201": {
+            "description": "The test base custom image 'create' operation was successfully enqueued. Check provisioningState to track execution status.",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "URL to query for status of the operation.",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "$ref": "#/definitions/CustomImageResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "CustomImageCreate": {
+            "$ref": "./examples/CustomImageCreate.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Test Base Custom Image"
+        ],
+        "operationId": "CustomImages_Delete",
+        "description": "Deletes a test base custom image.",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
```

Comment:
I'm not sure you need this.

Reply:
This is needed as it's a LRO to delete a custom image. To delete a custom image we need to delete the corresponding  image store so it takes time.

Reply:
Sorry, let me be more specific, I'm doubting that you need to specify  "final-state-via": "azure-async-operation"

Reply:
Its not part of the changes though, so we can skip it.